receiver/varnishreceiver/config.go

```diff
@@ -27,15 +28,21 @@ import (
 type Config struct {
 	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
 	Metrics                                 metadata.MetricsSettings `mapstructure:"metrics"`
-	WorkingDir                              string                   `mapstructure:"working_dir"`
+	InstanceName                            string                   `mapstructure:"instance_name"`
 	ExecDir                                 string                   `mapstructure:"exec_dir"`
 }
 
 // Validate validates the config.
 func (c *Config) Validate() error {
-	if c.WorkingDir != "" {
-		if _, err := os.Stat(c.WorkingDir); err != nil {
-			return fmt.Errorf(`"working_dir" does not exists: %w`, err)
+	if c.InstanceName != "" {
+		if _, err := os.Stat(c.InstanceName); err != nil {
+			return fmt.Errorf(`"instance_name" does not exists: %w`, err)
+		} else {
+			lastIndex := strings.LastIndex(c.InstanceName, "/")
+			if lastIndex+1 == len(c.InstanceName) {
+				return fmt.Errorf(`"instance_name" has invalid trailing "/" %s`, c.InstanceName)
```

Comment:
Curious do you think we should just fix this or should it be an error? Also wondering if we can use the pathing stuff in stdlib to just give us a valid path.

Reply:
outdated. checking for trailing / is not needed now because os.stat will send an error catching that